lib/vagrant-libvirt/action/create_network_interfaces.rb

```diff
@@ -194,7 +194,7 @@ def call(env)
               # It's used for provisioning and it has to be available during provisioning,
               # ifdown command is not acceptable here.
               next if slot_number.zero?
-              next if options[:auto_config] === false
```

Comment:
I missed that this is actually incorrect, was picked up by the integration test for the private network luckily.

The reason is that it will skip when `auto_config` is falsey, when we only want to skip when explicitly disabled.